lightning/src/util/config.rs

```diff
@@ -264,13 +264,10 @@ impl Default for ChannelConfig {
 }
 
 impl_writeable_tlv_based!(ChannelConfig, {
-	(0, forwarding_fee_proportional_millionths, required),
```

Comment:
Note that we cannot remove `required` elements, as old clients will refuse to load data serialized by new clients (which we guarantee won't happen at least across a few releases). We instead need to write the default value.